Project.toml

```diff
@@ -20,7 +20,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 CEnum = "0.4, 0.5"
 ChainRulesCore = "1"
 EnzymeCore = "0.7.5"
-Enzyme_jll = "0.0.127"
+Enzyme_jll = "0.0.126"
```

Comment:
Why is this necessary?

Reply:
oh CI wasn't running at all without the other package registered, so i did this temporarily to see if it would run

Reply:
Maybe there was a delay in the registry?  [v0.0.127](https://github.com/JuliaRegistries/General/pull/109889) was registered over 24 hours ago.